### PR TITLE
s/uniq/distinct

### LINKED ISSUE #198: DEPRECATION WARNING: uniq is deprecated and will be removed from Rails 5.1 (use distinct instead).
app/controllers/courses_controller.rb:12

### PR DESCRIPTION
**Pull request by ajm188**
yay removing deprecation warnings. this closes #198, as `git grep uniq` now comes up empty

### CODE REVIEW

File: app/controllers/courses_controller.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def index
                .filter_by_semester(params[:semester])
                .filter_by_professor(params[:professor])
                .order_by_short_name
-               .uniq
+               .distinct
```
Comment (ajm188) [inline app/controllers/courses_controller.rb]: ugh if I must

**Comment by ajm188** [comment]
also, that first commit failed a build due to flakiness

**Comment by matthewbentley** [comment]
Looks good :+1:

**Comment by StephHippo** [comment]
:ship: :it:

**Comment by ajm188** [comment]
It is flakiness. See [this commit](https://github.com/hacsoc/the_jolly_advisor/commit/d37bef09c60e250f54048af5405d75278219e8b9) for a description of the flake.

I am restarting the build now. I missed your comment b/c I was drowning in notifications at the time. My bad.

**Comment by ajm188** [comment]
Passed. Merging.

**Pull request merged**